cylc/flow/xtriggers/xrandom.py

```diff
@@ -45,6 +45,19 @@ def xrandom(
             Used to allow users to specialize the trigger with extra
             parameters.
 
+    Returns:
+        tuple: (satisfied, results)
+
+        satisfied:
+            True if ``satisfied`` else ``False``.
+        results:
+            A dictionary containing the following keys:
+
+            ``COLOR``
+                A random colour (e.g. red, orange, ...).
+            ``SIZE``
+                A random size (e.g. small, medium, ...).
+
```

Comment:
For some reason, napoleon doesn't like Returns coming after Examples, this seems to result in duplicate params/returns sections:

![Screenshot from 2024-04-24 11-04-04](https://github.com/cylc/cylc-flow/assets/16705946/48eef964-ebbe-4d00-a331-31fecada053f)
![Screenshot from 2024-04-24 11-04-15](https://github.com/cylc/cylc-flow/assets/16705946/d1eaf846-5352-47f5-b86a-c94c9849c252)